### DISCUSSION TITLE
How to Enable CDN (DO Spaces S3)

### DISCUSSION THREAD

**Discussion by noob8boi** [pocketbase/pocketbase#5689] [Q&A]
Hi, I have come across a problem where I am unable to add CDN endpoint for s3 storage in pocketbase. I checked other posts and I believe it is still not possible? The only way is to implement a custom solution which will check my DO Space CDN cache? 

Any sort of suggestions will be really helpful.

**Comment by thangngoc89** [comment]
I don’t know how DO Space CDN works but if it’s like other S3-compatible service then you could adapt my solution in the discussion at https://github.com/pocketbase/pocketbase/discussions/4871
